src/src/Helpers/SystemVariables.cpp

```diff
@@ -196,13 +198,13 @@ void SystemVariables::parseSystemVariables(String& s, boolean useURLencode)
 
   const int v_index = s.indexOf("%v");
 
-  if ((v_index != -1) && isDigit(s[v_index + 2])) {
-    for (byte i = 0; i < CUSTOM_VARS_MAX; ++i) {
-      String key = "%v" + String(i + 1) + '%';
-
+  if ((v_index != -1)) {
+    unsigned int i;
+    if (validUIntFromString(s.substring(v_index + 2), i)) {
+      String key = "%v" + String(i) + '%';
       if (s.indexOf(key) != -1) {
         const bool trimTralingZeros = true;
```

Comment:
That one reads as `trimTralala zeros` 😛 suggestion: `trimTrailingZeros` (not sure if I saw this one earlier or it's also somewhere else, but I'm sure I saw it before)

Reply:
Ah had to read it twice to know what you meant.
And it is copy/paste, so I will fix it in both places :)